### DISCUSSION TITLE
"orin"

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2976] [Q&A] [answered]
There is a repo that is called "orin" so i want to know what this could be... some old version of TShock possibly?

**Comment by Arthri** [comment]
Did you mean Orion?

**Comment by ghost** [comment]
yes...

**Comment by Arthri** [comment, marked as answer]
Orion was intended to be the future version of TSAPI(Plugin API) + TShock. It was a complete rewrite from the ground up, however, development has stopped and it was not finished completely
